app/services/data/get-capacity-breakdown.js

```diff
@@ -0,0 +1,29 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+const orgUnitFinder = require('../helpers/org-unit-finder')
+
+module.exports = function (id, type) {
+  var orgUnit = orgUnitFinder('name', type)
+
+  var table = orgUnit.capacityBreakdownView
+  var whereObject = {}
+
+  if (id !== undefined) {
+    whereObject.id = id
+  }
```

Comment:
I don't think the `whereObject` is required here as the `id` will come from the path param and therefore always be present.

Reply:
For national the view which retrieves the capacity graph data (`00104_national_capacity_view.js`) does not insert an id.  So id is set as undefined in the view for national.  
Should I change the graph view so it retrieves an id and change this behaviour?

Reply:
aah right, I was thinking this was querying each child record individually, in other words - I wasn't really thinking. That makes sense, you want all of the records in the national level's view and therefore don't need to filter. Ignore me :)

Reply:
I think most of our views don't insert an id at national level, a work around would be to add `select 0 AS id` to all national views, then we could just pass the url id all the way down.

Reply:
That sounds like a good solution and it would be nice to have a consistent way of accessing each organisational level but for now this is how it's done everywhere else so I would leave it.